helm/kvm-operator/templates/secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ include "resource.default.namespace" . }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+data:
+  secret.yml: {{ printf "service: {registry: {dockerhubToken: %q}}" .Values.Installation.V1.Secret.Registry.Dockerhub.Token | b64enc | quote }}
```

Comment:
@corest shouldn't we have something like this here?

```
  secret.yaml: {{ .Values.Installation.V1.Secret.KVMOperator.SecretYaml | b64enc | quote }}
```

Reply:
Talked with @corest. That should be a separate file. To be on par with azure-operator it could be named `dockerhub-secret.yaml:`